src/libertem/analysis/gridmatching.py

```diff
@@ -223,7 +223,7 @@ def _match_all(self, point_selection: PointSelection, zero, a, b):
         scaled_diffs = diffs / (np.maximum(1, np.abs(indices))**0.5)
         errors = np.linalg.norm(scaled_diffs, axis=1)
         matched_selector = errors < self.tolerance
-        matched_indices = rounded[matched_selector].astype(np.int)
+        matched_indices = rounded[matched_selector].astype(int)
```

Comment:
Does it maybe make sense to explicitly use a sized `dtype` like `np.int64`? Just using `int` means the reader needs to know the conversion rules of `numpy`, which I also would have to look up.

For the `bool` cases etc. it should not be a problem, as they are unambiguous.

Reply:
> Does it maybe make sense to explicitly use a sized `dtype` like `np.int64`?

In cases where the number can get very high it would make sense to explicitly use int64. In this particular case it doesn't make a difference as long as it is not int8. :-)

Reply:
..also in all other cases where np.int was changed.